app/src/main/java/com/example/myapplicationwork/MainActivity.kt

```diff
@@ -4,10 +4,28 @@ import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 
 
-class MainActivity : AppCompatActivity() {
+class MainActivity : AppCompatActivity() , FragmentMoviesList.TransactionsFragmentClicks {
+
+    private val moviesListFragment = FragmentMoviesList().apply { setClickListener(this@MainActivity) };
+    private val moviesDetailsFragment = FragmentMoviesDetails();
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        if(savedInstanceState == null)
+            supportFragmentManager.beginTransaction()
```

Comment:
not need to add movie list fragment from code
you can add inside activity_layout file 
[source](https://developer.android.com/guide/fragments/create#add-xml)